sql/src/main/java/io/crate/operation/scalar/cast/CastFunctionResolver.java

```diff
@@ -67,4 +71,7 @@ public static FunctionInfo functionInfo(DataType dataType, DataType returnType)
         return new FunctionInfo(new FunctionIdent(functionName, ImmutableList.of(dataType)), returnType);
     }
 
+    public static String getCastFunctionName(DataType dataType) {
```

Comment:
is this only used in tests? 
Can this also be package private? If that doesn't work I'd rather use hardcoded functionNames in the test or if you don't like that maybe add constants under a public static class Names like we do it to expose columns in the system TableInfo classes.